### PR TITLE
Update Amazon Search extension

### PR DESCRIPTION
**Pull request by xilopaint**
## Description

- Added recent search history.
- Improved empty state layout.

## Screenshot

Screenshot 2024-02-25 at 02 20 28
Screenshot 2024-02-25 at 10 32 34

## Checklist

- [x] I read the [extension guidelines](https://developers.raycast.com/basics/prepare-an-extension-for-store)
- [x] I read the [documentation about publishing](https://developers.raycast.com/basics/publish-an-extension)
- [x] I ran `npm run build` and [tested this distribution build in Raycast](https://developers.raycast.com/basics/prepare-an-extension-for-store#metadata-and-configuration)
- [x] I checked that files in the `assets` folder are used by the extension itself
- [x] I checked that assets used by the `README` are placed outside of the `metadata` folder

### CODE REVIEW

File: extensions/amazon-search/src/index.tsx [hunk 1]
```diff
@@ -49,24 +52,64 @@ export default function Command() {
...
+      {searchText.length === 0 && (
+        <List.Section title="Recent Searches">
+          {recentSearches.map((item, index) => (
+            <List.Item
+              key={index}
+              title={item}
+              actions={
+                <ActionPanel>
+                  <Action.OpenInBrowser url={`https://www.amazon.${tld}/s?k=${encodeURIComponent(item)}`} />
+                  <Action title="Remove Search Item" onAction={() => handleRemoveSearchItem(item)} />
```
Comment (pernielsentikaer) [inline extensions/amazon-search/src/index.tsx:100]: Maybe add `style={Action.Style.Destructive}`

File: extensions/amazon-search/src/index.tsx [hunk 2]
```diff
@@ -49,24 +52,64 @@ export default function Command() {
...
+                  <Action title="Clear Search History" onAction={handleClearSearchHistory} />
```
Comment (pernielsentikaer) [inline extensions/amazon-search/src/index.tsx:101]: Maybe add `style={Action.Style.Destructive}` and confirmAlert?

**Review by pernielsentikaer — CHANGES_REQUESTED** [review]
Hi 👋 

Thanks for your contribution 💪 

I have now tested your extension, and I have some feedback ready for you 🙂

I'm looking forward to testing this extension again 🔥 

Request a new review when you are ready. Feel free to contact me here or at [Slack](https://www.raycast.com/community) if you have any questions.

File: extensions/amazon-search/src/index.tsx (hunk 1 above)
Comment (xilopaint) [reply extensions/amazon-search/src/index.tsx]: I was not familiar with `Action.Style.Destructive`. Thanks for bringing it to my attention.

Done.

File: extensions/amazon-search/src/index.tsx (hunk 2 above)
Comment (xilopaint) [reply extensions/amazon-search/src/index.tsx]: Done.

**Comment by xilopaint** [comment]
It's ready @pernielsentikaer.

File: extensions/amazon-search/package.json [hunk 3]
```diff
@@ -11,7 +11,7 @@
   "license": "MIT",
   "commands": [
     {
-      "name": "index",
+      "name": "search-amazon",
```
Comment (pernielsentikaer) [inline extensions/amazon-search/package.json:14]: Can we keep the old name? hotkey, alias and ranking it bound to this identifier inside the extension 🙂

Comment (xilopaint) [reply extensions/amazon-search/package.json]: Done.

**Review by pernielsentikaer — APPROVED** [review]
Hi 👋 

Looks good to me, approved 🔥

**Pull request merged**